code/modules/mob/living/simple_animal/bot/medbot.dm

```diff
@@ -660,15 +671,28 @@ GLOBAL_VAR(medibot_unique_id_gen)
 						speak(message,radio_channel)
 						playsound(src, messagevoice[message], 50)
 						COOLDOWN_START(src, declare_cooldown, 10 SECONDS)
-			if(MEDIBOT_TREAT_SUCK)
+					C.visible_message("<span class='danger'>[src] injects [patient] with its syringe!</span>", \
+					"<span class='userdanger'>[src] injects you with its syringe!</span>")
+			if(MEDBOT_TREAT_BANDAGE)
+				var/mob/living/carbon/human/H = C
+				if(!H.bleed_rate)
+					to_chat(src, "<span class='warning'>[H] isn't bleeding!</span>")
+					return
+				if(H.bleedsuppress) //so you can't stack bleed suppression
+					to_chat(src, "<span class='warning'>[H]'s bleeding is already bandaged!</span>")
+					return
+				H.suppress_bloodloss(900) //half as good as a regular medical gauze
```

Comment:
This will become cauterise_wounds (which seals wounds) and suppress_bloodloss (which bandages wounds and keeps the status effect). With the new bleeding, the value will be between 0.5 to 2 probably, since it depends on the bleed rate rather than the time.

Reply:
probably shouldn't cauterise the wound, minor bandaging should be good enough? I've set it to use the same thing improvised gauze uses, could have it also be scaled with research efficiency?